test_merge_tree_s3_failover/test.py::test_move_failover flaky

```
>  TTL dt + INTERVAL 3 SECOND TO VOLUME 'external'

>       node.query(
            "INSERT INTO s3_failover_test VALUES (now() - 2, 0, 'data'), (now() - 2, 1, 'data')"
        )

E           Code: 499. DB::Exception: Received from 172.16.10.6:9000. DB::Exception: Message: Unable to parse ExceptionName: ExpectedError Message: Expected Error, Key: data/rwb/cnwzupfsxvgqqpujmpvatkwoorjio, Bucket: root, Object size: 1, WithPool: true. Stack trace:
```

https://s3.amazonaws.com/clickhouse-test-reports/46042/e1f345cc6b91b53553b5695a81557470bc85dd10/integration_tests__asan__[3/6].html

Here is only 1s gap until inserted data is going to be moved by ttl rule. That is bad, because time here is used with 1s resolution. Next second is happening faster that 1000ms usually. 
The new parts is legally moving to s3 at insert.
Here should be a larger gap.

--- a/tests/integration/test_merge_tree_s3_failover/test.py
+++ b/tests/integration/test_merge_tree_s3_failover/test.py
@@ -182,7 +182,7 @@ def test_move_failover(cluster):
             data String
         ) ENGINE=MergeTree()
         ORDER BY id
-        TTL dt + INTERVAL 3 SECOND TO VOLUME 'external'
+        TTL dt + INTERVAL 4 SECOND TO VOLUME 'external'
         SETTINGS storage_policy='s3_cold'
         """
     )
@@ -191,7 +191,7 @@ def test_move_failover(cluster):
     fail_request(cluster, 1)
 
     node.query(
-        "INSERT INTO s3_failover_test VALUES (now() - 2, 0, 'data'), (now() - 2, 1, 'data')"
+        "INSERT INTO s3_failover_test VALUES (now() - 1, 0, 'data'), (now() - 1, 1, 'data')"
     )
 
     # Wait for part move to S3.

